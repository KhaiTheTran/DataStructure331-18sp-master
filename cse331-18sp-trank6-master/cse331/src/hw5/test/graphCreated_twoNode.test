# Test graphCreated_twoNode

# Create a graph
CreateGraph graphCreated_twoNode

# Add a node
AddNode graphCreated_twoNode xx
AddNode graphCreated_twoNode yy

# Print the nodes in the graph and the outgoing edges from xx,
# and outgoing node yy
ListNodes graphCreated_twoNode
ListChildren graphCreated_twoNode xx
ListChildren graphCreated_twoNode yy