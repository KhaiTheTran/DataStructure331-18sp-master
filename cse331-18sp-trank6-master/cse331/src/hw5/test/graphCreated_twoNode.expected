# Test graphCreated_twoNode

# Create a graph
created graph graphCreated_twoNode

# Add a node
added node xx to graphCreated_twoNode
added node yy to graphCreated_twoNode

# Print the nodes in the graph and the outgoing edges from xx,
# and outgoing node yy
graphCreated_twoNode contains: xx yy
the children of xx in graphCreated_twoNode are:
the children of yy in graphCreated_twoNode are: